test/config_test/config_test.cc

```diff
@@ -61,12 +61,6 @@ class ConfigTest {
     ON_CALL(server_, sslContextManager()).WillByDefault(ReturnRef(ssl_context_manager_));
     ON_CALL(server_.api_, fileSystem()).WillByDefault(ReturnRef(file_system_));
     ON_CALL(server_.api_, randomGenerator()).WillByDefault(ReturnRef(random_));
-    ON_CALL(file_system_, fileReadToEnd(StrEq("/etc/envoy/lightstep_access_token")))
-        .WillByDefault(Return("access_token"));
-    ON_CALL(file_system_, fileReadToEnd(StrNe("/etc/envoy/lightstep_access_token")))
-        .WillByDefault(Invoke([&](const std::string& file) -> std::string {
-          return api_->fileSystem().fileReadToEnd(file);
-        }));
```

Comment:
I am wondering if these lines being removed somehow tickled other config tests into failing?

Reply:
yeah - the first of those says "if I am asked to read the lightstep access token return "access_token"
the second says "if I am asked to read something other than the lightstep access token, return the file contents

If you can reinstate the latter from
ON_CALL(file_system_, fileReadToEnd(StrNe("/etc/envoy/lightstep_access_token"))),,,,
to 
ON_CALL(file_system_, fileReadToEnd(_))
        .WillByDefault(Invoke([&](const std::string& file) -> std::string {
          return api_->fileSystem().fileReadToEnd(file);
        }));
you should be good!

Reply:
Thank you @alyssawilk!